src/org/parosproxy/paros/network/ConnectionParam.java

```diff
@@ -126,7 +134,7 @@
 	
 	// ZAP: Added prompt option and timeout
 	private boolean proxyChainPrompt = false;
-	private int timeoutInSecs = 120;
+	private int timeoutInSecs = DEFAULT_TIMEOUT;

```

Comment:
It should be used 20, right? (That was the value used when reading from configuration file.)